<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile analyzer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>985</width>
     <height>577</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="ver_tree">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="animated">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="headerShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">percentage (global)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>percentage (local)</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hor_tools">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="ver_start">
        <item>
         <widget class="QSlider" name="slider_start_ts">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hor_min_start">
          <item>
           <widget class="QLineEdit" name="min_ts">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="start_ts">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="ver_end">
        <item>
         <widget class="QSlider" name="slider_end_ts">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hor_max_end">
          <item>
           <widget class="QLineEdit" name="end_ts"/>
          </item>
          <item>
           <widget class="QLineEdit" name="max_ts">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="ver_upd">
        <item>
         <widget class="QPushButton" name="button_update">
          <property name="text">
           <string>update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_auto_update">
          <property name="text">
           <string>auto update</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="whole_time">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="reverse_call_stack">
      <property name="text">
       <string>Reverse call stack</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>&amp;menu</string>
    </property>
    <addaction name="actionOpen_DB_file"/>
    <addaction name="actionExpand_first_level"/>
    <addaction name="actionCollapse_all"/>
    <addaction name="actionReverse_call_stack"/>
    <addaction name="actionFind"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_DB_file">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;open DB file</string>
   </property>
  </action>
  <action name="actionExpand_first_level">
   <property name="text">
    <string>&amp;expand next level</string>
   </property>
  </action>
  <action name="actionCollapse_all">
   <property name="text">
    <string>&amp;collapse next level</string>
   </property>
  </action>
  <action name="actionReverse_call_stack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;reverse call stack</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>find</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
